packages/pages/docs/api/pages.clientserverrendertemplates.iscustomrendertemplate.md

```diff
@@ -0,0 +1,13 @@
+<!-- Do not edit this file. It is automatically generated by API Documenter. -->
+
+[Home](./index.md) &gt; [@yext/pages](./pages.md) &gt; [ClientServerRenderTemplates](./pages.clientserverrendertemplates.md) &gt; [isCustomRenderTemplate](./pages.clientserverrendertemplates.iscustomrendertemplate.md)
+
+## ClientServerRenderTemplates.isCustomRenderTemplate property
+
+If the render templates are user-defined or built-in
+
+**Signature:**
+
+```typescript
+isCustomRenderTemplate: boolean;
```

Comment:
Some of these generated files should not be public. Anything internal to PagesJS and not external functions a user could import should not be documented.

Reply:
Agree. It makes docs for everything marked as public + any dependencies those functions have.

for example: 
```
// @public
export interface Site {
  /** some comment. */
  branchId: string;
  ...
  }
```

Then we'd have a .md file for Site as well as for branchId